src/main/java/com/deavensoft/paketomat/courier/CourierController.java

```diff
@@ -92,4 +91,12 @@ public List<Package> getPackagesForCourier(@PathVariable(name = "city") String c
         log.info("List with packages that courier has to deliver are returned");
         return courierService.getPackagesForCourier(city);
     }
+
+    @GetMapping(path = "/getNotPickedUpPackages/")
+    @Operation(summary = "Get not picked up packages", description = "Get packages that not picked up by user")
+    @ApiResponse(responseCode = "200", description = "All packages that not picked up by user will be returned.")
+    public List<Package> getNotPickedUpPackages() throws PaketomatException {
```

Comment:
Replace `List<Package>` to `List<PackageDto>`

Reply:
Done.